lib/azkaban_scheduler/client.rb

```diff
@@ -5,8 +5,9 @@ module AzkabanScheduler
   class Client
     attr_reader :http
 
-    def initialize(url, http_options = {})
+    def initialize(url, http_options = {}, client_headers=nil)
```

Comment:
Might be better to default to an empty hash, especially because you're calling `.merge` below.